q2_phylogeny/plugin_setup.py

```diff
@@ -60,6 +62,72 @@
     citations=[citations['price2010fasttree']]
 )
 
+plugin.methods.register_function(
+    function=q2_phylogeny.raxml,
+    inputs={
+            'alignment': FeatureData[AlignedSequence]},
+    parameters={
+            'seed': Int,
+            'n_threads': Int % Range(1, None),
+            'substitution_model': Str % Choices(_RAXML_MODEL_OPT)},
+    outputs=[('tree', Phylogeny[Unrooted])],
+    input_descriptions={
+        'alignment': ('Aligned sequences to be used for phylogenetic '
+                      'reconstruction.'),
+    },
+    parameter_descriptions={
+        'n_threads': ('The number of threads to use for multithreaded '
+                      'processing. Using more than one thread '
+                      'will enable the PTHREADS version of RAxML.'),
+        'substitution_model': ('Model of Nucleotide Substitution'),
+        'seed': ('Random number seed for the parsimony starting tree. '
+                 'This allows you to reproduce tree results. '
+                 'If not supplied then one will be randomly chosen. ')},
+    output_descriptions={'tree': 'The resulting phylogenetic tree.'},
+    name='Construct a phylogenetic tree with RAxML.',
+    description=("Construct a phylogenetic tree with RAxML."),
+    citations=[citations['Stamatakis2014raxml']]
+)
+
+plugin.methods.register_function(
+    function=q2_phylogeny.raxml_rapid_bootstrap,
+    inputs={
+            'alignment': FeatureData[AlignedSequence]},
+    parameters={
+            'seed': Int,
+            'rapid_bootstrap_seed': Int,
+            'bootstrap_replicates': Int,
+            'n_threads': Int % Range(1, None),
+            'substitution_model': Str % Choices(_RAXML_MODEL_OPT)},
+    outputs=[('tree', Phylogeny[Unrooted])],
+    input_descriptions={
+        'alignment': ('Aligned sequences to be used for phylogenetic '
+                      'reconstruction.'),
+    },
+    parameter_descriptions={
+        'n_threads': ('The number of threads to use for multithreaded '
+                      'processing. Using more than one thread '
+                      'will enable the PTHREADS version of RAxML.'),
+        'substitution_model': ('Model of Nucleotide Substitution'),
+        'seed': ('Random number seed for the parsimony starting tree. '
+                 'This allows you to reproduce tree results. '
+                 'If not supplied then one will be randomly chosen. '),
+        'rapid_bootstrap_seed': ('Specify a random seed for rapid '
```

Comment:
Can you tighten up the line breaks on for this description, it is a little hard to read in the source.

e.g.

```python
'rapid_bootstrap_seed': ('Specify a random seed for rapid '
                         'bootstrapping. This allows you to reproduce '
                         'rapid bootstrap results. If not supplied '
                         'then one will be randomly chosen.'),
```